index.html

```diff
@@ -28,6 +28,142 @@ <h1>Iheagwara<br>Augustine</h1>
         <button class="Start-collab">Start collaboration</button>
         </section>            
         <div class="image-1-wrapper"><div class="image-1"></div></div>
+        <section class="section-2">
+            <h1 class="projects">Projects</h1>
+            <div class="project-items-1">
+                <img src="images.svg\Rectangle 21 (19).svg" class="project-image" alt="">
+                <div class="languages">
+                    <h2>Project name goes here</h2>
+                    <ul class="H-R-J">
+                        <li class="Html-css">HTML/CSS</li>
+                        <li class="Ruby">RUBY ON RAILS</li>
+                        <li class="Javascript">JAVASCRIPT</li>
+                    </ul>
+                    <button class="see-this-project">
```

Comment:
<img width="736" alt="3" src="https://user-images.githubusercontent.com/77843050/212502341-40556d84-ac6b-447e-8f9e-4d5bf6976b46.png">

- [ ] Please play with the text font and uppercase to modify your design, the project name title is bold than the original one, and the technologies text are capitalized on the left, the template does not have those style, please fix that.